### PR TITLE
KK-156 | Accessibility improvements

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/domain/registration/form/RegistrationForm.tsx [hunk 1]
```diff
@@ -169,10 +169,11 @@ const RegistrationForm: FunctionComponent = () => {
                 </div>
                 <div className={styles.registrationGrayContainer}>
                   <Button
+                    aria-label={t('child.form.modal.add.label')}
                     className={styles.addNewChildButton}
                     onClick={() => setIsOpen(true)}
                   >
-                    <Icon src={addIcon} alt="Add child icon"></Icon>
+                    <Icon src={addIcon} alt=""></Icon>
```
Comment (midori0507) [inline src/domain/registration/form/RegistrationForm.tsx]:
```suggestion
                    <Icon src={addIcon} alt="" />
```

File: src/domain/app/layout/PageWrapper.tsx [hunk 2]
```diff
@@ -28,12 +28,17 @@ const PageWrapper: FunctionComponent<{
       ? t(description)
       : t('homePage.hero.descriptionText');
 
+  const path = window.location.pathname.replace(`/${lang}/`, '');
```
[the remark this answers is not in the document]
Comment (badrange) [reply src/domain/app/layout/PageWrapper.tsx]: Good point! That doesn't work for us right now, I added a TODO about fixing this.

File: src/domain/app/layout/__tests__/__snapshots__/PageWrapper.test.tsx.snap [hunk 3]
```diff
@@ -18,6 +18,21 @@ exports[`renders snapshot correctly 1`] = `
...
+      hrefLang="fi"
+      rel="alternate"
+    />
+    <link
+      href="/sv//"
+      hrefLang="sv"
+      rel="alternate"
+    />
+    <link
+      href="/en//"
```
[the remark this answers is not in the document]
Comment (badrange) [reply src/domain/app/layout/__tests__/__snapshots__/PageWrapper.test.tsx.snap]: Added a TODO for this too, to be fixe when we start using updateLocaleParam

File: src/domain/home/instructions/HomeInstructions.tsx [hunk 4]
```diff
@@ -15,23 +15,15 @@ const HomeInstructions: React.FunctionComponent = props => {
...
+            <Icon src={homeTicketIcon} className={styles.icon} alt="" />
             <p>{t('home.instructions.icon.ticket.text')}</p>
           </div>
           <div className={styles.iconBox}>
-            <Icon
-              src={homeTheaterIcon}
-              className={styles.icon}
-              alt="ticket icon"
-            />
+            <Icon src={homeTheaterIcon} className={styles.icon} alt="" />
```
[the remark this answers is not in the document]
Comment (badrange) [reply src/domain/home/instructions/HomeInstructions.tsx]: Good idea!!

File: src/common/components/dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -38,16 +40,25 @@ const Dropdown: React.FunctionComponent<DropdownProps> = ({
...
   }, []);
 
+  const ariaLabel = isOpen
+    ? t('common.menu.closeMenuText')
+    : t('common.menu.openMenuText');
+
   return (
     <div className={styles.dropdownWrapper} {...rest} ref={ref}>
       <Button
+        aria-label={ariaLabel}
```
[the remark this answers is not in the document]
Comment (badrange) [reply src/common/components/dropdown/Dropdown.tsx]: I don’t think we should make aria-label required. We only need it for the few cases where the text of the button is ‘x’ or the button is an icon.

[the remark this answers is not in the document]
Comment (badrange) [reply src/common/components/dropdown/Dropdown.tsx]: Adding it as an optional prop

File: src/common/components/dropdown/Dropdown.tsx [hunk 6]
```diff
@@ -38,16 +40,25 @@ const Dropdown: React.FunctionComponent<DropdownProps> = ({
...
+        aria-expanded={isOpen}
         onClick={() => {
           toggleDropdown(!isOpen);
           options[0].onClick && options[0].onClick();
         }}
       >
         <span>{options[0].label}</span>
-        <Icon src={options[0].icon || angleDownIcon} alt="Dropdown icon" />
+        <Icon
+          src={options[0].icon || angleDownIcon}
```
[the remark this answers is not in the document]
Comment (badrange) [reply src/common/components/dropdown/Dropdown.tsx]: All right!

**Pull request merged**